<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>483</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>423</width>
    <height>483</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Date Center</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Datas.qrc">
    <normaloff>:/Imagens/Datas.ico</normaloff>:/Imagens/Datas.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>281</width>
      <height>211</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Calendário Mensal</string>
    </property>
    <property name="gridVisible">
     <bool>false</bool>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="navigationBarVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOk">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Executa ação selecionada</string>
    </property>
    <property name="text">
     <string>&amp;Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Sai da aplicação</string>
    </property>
    <property name="text">
     <string>&amp;Cancela</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grbAcoes">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Ações</string>
    </property>
    <widget class="QRadioButton" name="rbtFeriados">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>82</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Mostra Feriados</string>
     </property>
     <property name="text">
      <string>Feriados</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtConversao">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Conversão entre Datas (Civil &lt;-&gt; Hebraica)</string>
     </property>
     <property name="text">
      <string>Conversão</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtDiferenca">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Diferença entre Datas</string>
     </property>
     <property name="text">
      <string>Diferença</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grbOpcoes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Opções</string>
    </property>
    <widget class="QGroupBox" name="grbTipoFeriado">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Tipo dos Feriados</string>
     </property>
     <widget class="QRadioButton" name="rbtCivis">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>28</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Mostra Feriados Civis</string>
      </property>
      <property name="text">
       <string>Civis</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtHebraicos">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>28</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Mostra Fertiados Hebraicos</string>
      </property>
      <property name="text">
       <string>Hebraicos</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="chkMarcar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Marcar</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tblFeriados">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>401</width>
      <height>151</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>A Data indica a VÉSPERA do Feriado (Pôr-do-Sol)</string>
    </property>
    <property name="rowCount">
     <number>12</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>Datas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Feriado</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Luach</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>25/09/2014</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>Rosh Hashaná</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>01/07/5775</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>04/10/2014</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>Yom Kippur</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>10/07/5775</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>09/10/2014</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>Sukkot</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>15/07/5775</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>17/12/2014</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>Chanukkah</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>25/09/5775</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>04/02/2015</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>T'u B'Shvat</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>15/11/5775</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>04/03/2015</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>Taanit Ester</string>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>13/12/5775</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblDataCivil">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Civil: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtDataCivil">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblDataHebraica">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Hebraica:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtDataHebraica">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblDataConvertida">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Convertida:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtDataConvertida">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>423</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>calendarWidget</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>rbtFeriados</tabstop>
  <tabstop>rbtConversao</tabstop>
  <tabstop>rbtDiferenca</tabstop>
  <tabstop>rbtCivis</tabstop>
  <tabstop>rbtHebraicos</tabstop>
  <tabstop>edtDataCivil</tabstop>
  <tabstop>edtDataHebraica</tabstop>
  <tabstop>edtDataConvertida</tabstop>
  <tabstop>tblFeriados</tabstop>
  <tabstop>chkMarcar</tabstop>
 </tabstops>
 <resources>
  <include location="Datas.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtHebraicos</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnHebraicos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calendarWidget</sender>
   <signal>currentPageChanged(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>MudandoMes(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtCivis</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnCivis()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtFeriados</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnFeriados()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtConversao</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnConversao()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtDiferenca</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnDiferenca()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtDataCivil</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnEditFinishDtCivil()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtDataHebraica</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnEditFinishDtHebraica()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnHebraicos()</slot>
  <slot>MudandoMes(int,int)</slot>
  <slot>OnCivis()</slot>
  <slot>OnFeriados()</slot>
  <slot>OnConversao()</slot>
  <slot>OnDiferenca()</slot>
  <slot>OnOk()</slot>
 </slots>
</ui>
